tests/network-tests/src/misc/postMigrationAssertionsFlow.ts

```diff
@@ -0,0 +1,32 @@
+import { assert } from 'chai'
+import { FlowProps } from '../Flow'
+import { extendDebug } from '../Debugger'
+
+export default async function postMigrationAssertions({ api }: FlowProps): Promise<void> {
+    const debug = extendDebug('flow:postMigrationAssertions')
+    debug('Started')
+
+    // check that counters haven't been re-set        
+    debug('Checking that Video, Channel, Categories  counters have not been re-set')
+
+    const nextVideoCategoryId = await api.query.content.nextVideoCategoryId()
+    const nextVideoId = await api.query.content.nextVideoId()
+    const nextChannelId = await api.query.content.nextChannelId()
+
+    const num_channels = await api.getNumberOfOutstandingChannels()
+    const num_videos = await api.getNumberOfOutstandingVideos()
+    const num_categories = await api.getNumberOfOutstandingVideoCategories()
+
+    assert(nextVideoCategoryId.toNumber() > 1);
```

Comment:
These assertions are failing because `.toNumber()` is returning a `BN` type. so you want:
```suggestion
    assert(nextVideoCategoryId.toNumber().gt(1));
```